compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -143,6 +143,7 @@ class OperationExporter final : public luci::CircleNodeMutableVisitor<void>,
   void visit(luci::CircleOutputDummy *) final {}
   void visit(luci::CircleOutputExclude *) final {}
   // Virtual for multiple-outputs
+  void visit(luci::CircleCustomOut *) final {}
```

Comment:
You may also have to revise `void OperationExporter::visit(luci::CircleCustom *node)` in this file